### ISSUE TITLE
Database Info Status Bug - This application is not running with a Postgres database backend

### ISSUE THREAD

**Issue by tomsinclair94** [TandoorRecipes/recipes#2752]
### Tandoor Version

1.5.8

### Setup

Docker / Docker-Compose

### Reverse Proxy

No reverse proxy

### Bug description

It appears that the system status page has a bug with the "Database Info" section. Despite there being no reason why it is not using a PostgreSQL DB as the backend, the Info/Warning message is displayed:

```
Database Info
This application is not running with a Postgres database backend. This is ok but not recommended as some features only work with postgres databases.
```

This showed as "Everything fine" in 1.5.6 but after upgrading to 1.5.8, it now shows the above. This is using PostgreSQL 15.5 but also does it with PostgreSQL 16.

**Comment by smilerz** [comment]
This is fixed in #2730

**Comment by vabene1111** [comment]
I should probably get it merged earlier...
